emcc.py

```diff
@@ -2512,6 +2515,9 @@ def get_full_import_name(name):
 
     settings.ASYNCIFY_IMPORTS = [get_full_import_name(i) for i in settings.ASYNCIFY_IMPORTS]
 
+    if settings.ASYNCIFY == 2 and not settings.EXIT_RUNTIME:
+      diagnostics.warning('experimental', 'ASYNCIFY with stack switching is only tested with EXIT_RUNTIME atm')
```

Comment:
How about just always emitting the warning when `ASYNCIFY=2` is specified?   It is experimental right now for all uses, right?

Reply:
Sounds good, I'll warn all the time. I'll also add an error here with exit runtime, as it looks like that is just broken atm in d8 (which is the only place we can test so far).